Documentation/CRP.md

```diff
@@ -0,0 +1,103 @@
+# CRP Implementation Details
```

Comment:
Considering that you mention "Anything specific to Babylon (especially process and specific commands) ➡️ in the playbook" wouldn't be better to have it in a playbook?

As of the content of this doc I thought it would be more about how this command should work and why (business rules) rather than implementation details with methods names etc. I feel this would me more valuable and implementation details documentation belongs more to the code comments.

Reply:
Business rules are documented in the other PR in the playbook, which indeed document the usage.

I can move implementation details to the playbook too if it makes more sense. Would also allow to reduce that document by removing anything that's trivial when just reading the code

(My initial understanding was that everything about technical implementation should be documented in repo so that anyone who'd need to maintain the CRP code in my absence would have all they need documented; even though I agree that code is pretty much documented or clear already, but that was one of the AC of the ticket, which is why I wrote it that way)